tests/test_smoketest.py

```diff
@@ -108,14 +110,20 @@ def smoketest(self, starting_replica, checkout_bucket, test_bucket):
             f" | jq .biomaterial_core.biomaterial_id=env.biomaterial_id"
             f" | sponge {self.bundle_dir}/cell_suspension_0.json",
             env=dict(os.environ, biomaterial_id=biomaterial_id))
-        query = {'query': {'match': {'files.cell_suspension_json.biomaterial_core.biomaterial_id': biomaterial_id}}}
+
+        # creating both tombstone subscriptions and a bundle subscriptions
+        queries = [
+            {'query': {'match': {'files.cell_suspension_json.biomaterial_core.biomaterial_id': biomaterial_id}}},
+            tombstone_query
```

Comment:
Consider inlining `tombstone_query` here, or moving the normal query into `tests.sample_search_queries`.

Reply:
inlined tombstone query